pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -32,16 +34,28 @@ const (
 	mountNamespace = "/proc/1/ns/mnt"
 )
 
+// GetHostNamespacePath - return the path of the parent process mount namespace
```

Comment:
I would sleep little bit better if we change this PR to something like this:
1. By default use `/proc/1` as host namespace path...
2. Add flag to virt-handler to switch the logic and dynamically detect parent Pid.

@naiming-zededa How does this sounds to you?

Reply:
Sure. This sounds ok to me. And do you have some examples of feature tag and being used in the virt-handler I can take a look to see how to add that.

Reply:
You would add flag [here](https://github.com/kubevirt/kubevirt/blob/main/cmd/virt-handler/virt-handler.go#L546) and propagate it down as you need.

Reply:
@xpivarc Hi, this whole scenario is unclear to me, in terms of what exactly are we going to support here and how will we test it?

Reply:
@enp0s3 I suggest by default to retain the code paths we have and add option to dynamically detect the host namespace. We don't have infra to test this either way but at least with this approach we know that default case is working. WDYT? 

Unit tests should be provided imho for this.
Even better would be some infra...

Reply:
@xpivarc +1
I also think that anyway we should test the "API" and not the implementation details. Indeed infra would be nice

Reply:
> You would add flag [here](https://github.com/kubevirt/kubevirt/blob/main/cmd/virt-handler/virt-handler.go#L546) and propagate it down as you need.

@xpivarc say i add another flag 'VirtInContainer' there and in the feature.yaml in the developerconfig, set the
' - VirtInContainer', and a function
func (config *ClusterConfig) VirtRunInContainerEnabled() bool {
	return config.isFeatureGateEnabled(VirtRunInContainer)
}

what will be the good way for functions inside 'package selinux' to access that function?
thanks.

Reply:
You don't need to add `func (config *ClusterConfig) VirtRunInContainerEnabled() bool {` we just need a local flag for the handler and propagate it. Be it by passing it through the functions or just setting it once one the handler startup (which should be enough).

Reply:
@xpivarc , @enp0s3 I have updated the change and pushed into this PR.
the developerConfig does not work, since it needs to apply the feature yaml later and by the time the pods has already be setup in terms of the namespace mounts, etc.
The only reasonable way is to pass in the ENV, and that is simple and does not involve with extra coding related to the config changes.
So, now with this update. Normal users working with kubevirt has no effect at all and their system should work exact as before, while allow our open-source EVE with kubeviurt extension to work.
Hope this resolves any outstanding issues for the PR. Please review. Thanks.